library/Garp/Model/Db/Snippet.php

```diff
@@ -21,7 +21,14 @@ public function fetchByIdentifier($identifier) {
         if ($result = $this->fetchRow($select)) {
             return $result;
         }
-        throw new Exception('Snippet not found: ' . $identifier);
+        if (!Zend_Registry::get('config')->snippets->ignoreMissing) {
+            throw new Exception('Snippet not found: ' . $identifier);
+        }
+        // Return fallback row, where text is set to $identifier, in order to provide some fallback.
```

Comment:
I do hope we're not getting a polluted database due to typos etc... but yeah, let's choose the path of least resistance.

Reply:
We won't! This all happens at runtime (or read-time), so the only effect would be a wrong string being used in the interface (namely the identifier instead of the actual value).